src/gen_dragonfruit/gd_dev_apogee_sp215.cpp

```diff
@@ -1,27 +1,41 @@
 #include "gd_dev_apogee_sp215.h"
 
 void gd_dev_apogee_sp215_open(void){
-    Wire.begin(9600);
+
+    /* Initiate Wire library and join I2C bus as slave */
+    Wire.begin();
+
+    /* Begin transmission to the I2C slave device with the given address */
+    Wire.beginTransmission(_DEV_ADDR_GD_APOGEE_SP215_);
+
+    /* Enable read bit and select 16-bit sample rate and select sampling rate
+    of 8 SPS. */
+    Wire.write(0x8C);
```

Comment:
You should change your comment to specify which bits you're changing, or use bit masking to make the configuration more understandable.